server/controllers/review.js

```diff
@@ -0,0 +1,24 @@
+const Review = require("../models/Review");
+
+// @route GET /reviews/all/:sitterId
+// @desc gets all reviews for sitter
+// @access Public
+exports.getAllReviews = async (req, res) => {
+  const reviews = await Review.find({ sitterId: req.params.sitterId });
+  res.json({ success: reviews });
+};
+
+// @route POST /reviews/create/:sitterId
+// @desc creates new review
+// @access Public
+exports.createReview = async (req, res) => {
+  const { clientId, rating, description } = req.body;
```

Comment:
I think it would be better if we send the `sitterId` through the request body and the `clientId` can be obtained using the request object - `req.user.id`